### PR TITLE
Adds p2 Physics for P

### PR DESCRIPTION
**Pull request by edmundquintero**

### CODE REVIEW

File: src/player/player.js [hunk 1]
```diff
@@ -17,26 +17,25 @@ export class Player extends Phaser.Sprite {
...
    * Setup sprite body settings
    */
   bodySetup () {
-    this.game.physics.enable([this], Phaser.Physics.ARCADE);
-    this.body.gravity.y = 20000;
-    this.body.bounce.y = 0.2;
-    this.body.collideWorldBounds = true;
+    // this.fixedRotation = true;
+    // this.damping = 0.2;
+    // this.collideWorldBounds = true;
```
Comment (edmundquintero) [inline src/player/player.js:38]: I know these are useless but they are reminders for  next steps for me to figure out how to apply specific Physics rules to objects.

**Comment by bentok** [comment]
Looking at this right now.

**Comment by edmundquintero** [comment]
Ok this showed up.

File: gulpfile.babel.js [hunk 2]
```diff
@@ -39,7 +39,8 @@ gulp.task('server', () => {
 
 // Copy images to dist
 gulp.task('copyImages', () => gulp.src('src/images/*.*')
-  .pipe(gulp.dest('dist/images')));
+  .pipe(gulp.dest('dist/images'))
```
Comment (bentok) [inline gulpfile.babel.js:42]: We don't want a semicolon here?

**Review by bentok — COMMENTED** [review]
One pending review comment.

**Review by bentok — APPROVED** [review]

**Pull request merged**